projects/docs/docs/contributors/code-reviews.md

```diff
@@ -59,6 +59,9 @@ must include documentation.
 Pull requests should add an entry to that file describing in one sentence what the change is about.
 They should link to the pull request and mention the author or authors of the changes.
 
-**✅ Tests:** Continuous integration must be happy with the changes.
+**🚦 Continuous integration:** Continuous integration must be happy with the changes.
 The pipelines are designed to bring an extra level of confidence and validate that the changes are right.
 A red CI blocks the merge of the pull request.
+
+**✅ Merging:** A pull request is ready to be merged once there are at least 2 approvals from [Tuist core members](core-team.md).
```

Comment:
Note that the path is [/contributors/core-team](https://docs.tuist.io/contributors/core-team)